packages/e2e-test-utils-playwright/package.json

```diff
@@ -1,7 +1,6 @@
 {
 	"name": "@wordpress/e2e-test-utils-playwright",
-	"version": "0.0.0",
-	"private": true,
+	"version": "0.1.0-prerelease",
```

Comment:
I'm unsure how this will work with Lerna and the release process, the release process is documented at:
- https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/release.md

I believe Lerna and the release process creates a commit to bump package versions in `package.json`

Potentially the version would need to be `"version": "0.1.0",` to allow Lerna to version bump correctly

Worst case scenario this package may require it be manually published to npm and retain `"private": true,`

The above version bump scenarios could be checked via https://github.com/WordPress/gutenberg/blob/3bb63e41723f9e986a0c43a03378305c4be13e7e/package.json#L288

Reply:
Prerelease versions like `0.1.0-prerelease` or `0.1.0-alpha` are [valid semantic versions](https://semver.org/#spec-item-9). I believe they are [supported](https://lerna.js.org/docs/features/version-and-publish#versioning) in lerna as well. But I don't know if it's supported in our current workflow though 😅 .

Reply:
@ntwb, is there a Lerna test config or dry run that could be used to validate how the version will change? (Or another project it could be safely tested on?)